### DISCUSSION TITLE
로깅에 들어갈 사용자 정보 조회 방법 논의

### DISCUSSION THREAD

**Discussion by choichanhyeok** [2022-Harmony/NewsCommunity-bFinal#135] [Ideas]
<img width="901" alt="5675845463" src="https://user-images.githubusercontent.com/68278903/179935889-5d0739da-194c-47e0-af34-f069cc42b97a.PNG">

- 현재 예진님이 servletPath를 로깅에 넣을 수 있는지 여부 물어봐, 위와 같이 적용시킴
- 위 로그에 요청한 사용자 정보가 들어가면 좋을 거 같은데, 이에 대한 방법 의논했으면 좋겠음
- ? 로그인하지 않은 사용자가 보내는 요청은 어떻게 처리할 것인가

**Comment by enyo9rt** [comment]
SecurityContextHolder.getContext().getAuthentication().getPrincipal()
이게 Object 객체인데 비로그인 이용자면 anonymousUser 로 나옵니다
로그인유저 .getName() 하면 String 값 나와요
